scripts/us_census/pep_srh/process.py

```diff
@@ -0,0 +1,1047 @@
+    df.insert(3, 'MEASUREMENT_METHOD', 'dcs:CensusPEPSurvey', True)
+    df.to_csv(_CODEDIR + PROCESS_AS_IS_DIR + 'state_consolidated_temp.csv',
+              header=False,
+              index=False,
+              mode='a')
+
+    df = pd.read_csv(_CODEDIR + PROCESS_AS_IS_DIR +
+                     'state_consolidated_temp.csv')
+    df.sort_values(by=['LOCATION', 'SV', 'YEAR'], inplace=True)
+    df.to_csv(_CODEDIR + PROCESS_AS_IS_DIR +
+              'state_consolidated_as_is_final.csv',
+              header=True,
+              index=False)
+
+    if os.path.exists(_CODEDIR + PROCESS_AS_IS_DIR \
+        + 'state_consolidated_temp.csv'):
+        os.remove(_CODEDIR + PROCESS_AS_IS_DIR + 'state_consolidated_temp.csv')
+
+    # Agg file processing
+    state_agg_files = [
+        _CODEDIR + PROCESS_AGG_DIR + '1980_1990/state/state_1980_1990.csv',
+        _CODEDIR + PROCESS_AGG_DIR + '1990_2000/state/state_1990_2000.csv',
+        _CODEDIR + PROCESS_AGG_DIR + '2000_2010/state/state_2000_2010.csv',
+        _CODEDIR + PROCESS_AGG_DIR + '2010_2020/state/state_2010_2020.csv'
+    ]
+
+    for file in state_agg_files:
+        df = pd.read_csv(file)
+        df = df.melt(id_vars=['YEAR', 'LOCATION'],
+                     var_name='SV',
+                     value_name='OBSERVATION')
+        df.replace({"SV": STAT_VAR_COL_MAPPING}, inplace=True)
+        df["SV"] = 'dcid:' + df["SV"]
+
+        if file == state_agg_files[0]:
+            df.to_csv(_CODEDIR + PROCESS_AGG_DIR +
+                      'state_consolidated_temp.csv',
+                      header=True,
+                      index=False)
+        else:
+            df.to_csv(_CODEDIR + PROCESS_AGG_DIR +
+                      'state_consolidated_temp.csv',
+                      header=False,
+                      index=False,
+                      mode='a')
+
+    df = pd.read_csv(_CODEDIR + PROCESS_AGG_DIR + 'state_consolidated_temp.csv')
+    df.sort_values(by=['LOCATION', 'SV', 'YEAR'], inplace=True)
+    df.insert(3, 'MEASUREMENT_METHOD', 'dcs:dcAggregate/CensusPEPSurvey', True)
+    df.to_csv(_CODEDIR + PROCESS_AGG_DIR + 'state_consolidated_agg_final.csv',
+              header=True,
+              index=False)
+
+    if os.path.exists(_CODEDIR + PROCESS_AGG_DIR \
+    + 'state_consolidated_temp.csv'):
+        os.remove(_CODEDIR + PROCESS_AGG_DIR + 'state_consolidated_temp.csv')
+
+
+def _consolidate_county_files():
+    '''
+    Consolidating all county level files into single file
+    At the same time only carrying data which is relevant to SV by SRH
+    This funtion consolidtes both as-is and agg inddividual state files
+    '''
+    input_file1 = _CODEDIR + PROCESS_AS_IS_DIR \
+        + '2000_2010/county/county_2000_2010.csv'
+    input_file2 = _CODEDIR + PROCESS_AS_IS_DIR \
+        + '2010_2020/county/county_2010_2020.csv'
+
+    county_file = []
+    county_file.append(input_file1)
+    county_file.append(input_file2)
+
+    for file in county_file:
```

Comment:
Can we put some more description around AS_IS and AGGREGATE processing, and take the common code out of control block? In general for all consolidate functions